docs/content/mobile/android/local-deploy.md

```diff
@@ -34,7 +40,7 @@ You need the Unity Remote app for this. See the [Unity documentation](https://do
 
 1. You should now see your Unity Editor's game view mirrored on your Android device.
 
-## Use your Android emulator {#emulator}
+## Connect to a local deployment using Android Emulator{#android-emulator}
```

Comment:
The whole page is about connecting to a local deployment - the key information we are looking for on this page is:
What you are using repeating "Connect to a local deployment" on every title is extraneous and makes the titles harder to scan.

If you want to make sure it's super clear what the instructions do for any links to this section, you can add a short sentence which describes what the section is about but, I am guessing you probably want to make sure users are prepared first so they need to read the top bit anyhow.
Potential sentence:
"Connect to a  local deployment using Android Emulator"  <- if we are going to go with it as a proper noun. "Android Emulator", the Android docs have an odd pattern of sometimes using a proper noun and sometimes not. :-) It sort of works, I think.

Reply:
fixed